compiler-rt/test/profile/instrprof-tmpdir.c

```diff
@@ -12,7 +12,8 @@
 // RUN: llvm-profdata show ./raw2.profraw | FileCheck %s -check-prefix TMPDIR
 //
 // Check that we fall back to the default path if TMPDIR is missing.
-// RUN: env -u TMPDIR LLVM_PROFILE_FILE="%%t/raw3.profraw" %run %t/binary 2>&1 | FileCheck %s -check-prefix MISSING
+// RUN: unset TMPDIR
```

Comment:
I see no sign that `unset` is implemented in the LIT built-in shell.

Reply:
You're right, `unset` does not seem to be implemented in the built-in shell,
and since windows doesn't have unset, I'm making this an aix-only change.
Thanks.